### PR TITLE
Loki: Add tests for explain section in builder and code editor

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:
Related to https://github.com/grafana/grafana/pull/50437. This PR adds tests for new explain section in builder mode and code mode.

### CODE REVIEW

File: public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilder.test.tsx [hunk 1]
```diff
@@ -17,8 +20,8 @@ const createDefaultProps = () => {
     {
       url: '',
       jsonData: {},
-      meta: {} as any,
-    } as any,
+      meta: {} as DataSourcePluginMeta,
+    } as DataSourceInstanceSettings,
```
Comment (itsmylife) [inline public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilder.test.tsx:24]: making betterer happy 🚀

File: public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderExplained.tsx [hunk 2]
```diff
@@ -10,6 +10,8 @@ import { lokiQueryModeller } from '../LokiQueryModeller';
 import { buildVisualQueryFromString } from '../parsing';
 import { LokiVisualQuery } from '../types';
 
+export const EXPLAIN_LABEL_FILTER_CONTENT = 'Fetch all log lines matching label filters.';
```
Comment (itsmylife) [inline public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderExplained.tsx:13]: this is a nice touch 👍

**Review by itsmylife — APPROVED** [review]
LGTM

File: public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilder.test.tsx (hunk 1 above)
Comment (ivanahuckova) [reply public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilder.test.tsx:24]: and @joshhunt 🙂

**Pull request merged**